src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Deadline extends Task {
+    protected FormatDate date;
+
+    /**
+     *
+     * @param description description of task
+     */
```

Comment:
On top of what phyliciachristel mentioned, maybe capitalise your descriptions and end them off with a full stop, as per the guidelines [here](url)?